test/e2e/storage/vsphere/vsphere_volume_node_poweroff.go

```diff
@@ -170,10 +168,9 @@ func waitForPodToFailover(client clientset.Interface, deployment *appsv1.Deploym
 
 	if err != nil {
 		if err == wait.ErrWaitTimeout {
-			framework.Logf("Time out after waiting for %v", timeout)
+			return "", fmt.Errorf("pod has not failed over after %v: %v", timeout, err)
```

Comment:
for all these tests where we check `err == wait.ErrWaitTimeout` why is it important to disambiguate this from the standard error case? Wont the `err` contain the information required to understand what kind of error it is, timeout or not

Reply:
The difference is whether the timeout duration gets added into the log. Only benefit is making the error output more readable, which is why I left it as-is. Example output would be,
```
pod has not failed over after 5m: timed out waiting for the condition
```
or
```
pod did not fail over from "node-1": Pod Not Found
```

Reply:
got it, change looks good then